cmake/projects/freetype/hunter.cmake

```diff
@@ -15,9 +15,9 @@ hunter_add_version(
     VERSION
     "2.6.2"
     URL
-    "http://sourceforge.net/projects/freetype/files/freetype2/2.6.2/freetype-2.6.2.tar.bz2/download"
+    "https://github.com/hunter-packages/freetype/archive/hunter.tar.gz"
```

Comment:
This file will change from time to time. For example if I add some commit to branch `hunter` then SHA1 of archive will change => download will fail with hash mismatch. You need to create release and add stable/release archive here.